File: app/src/main/java/com/example/karol/musicapp/MusicApp.java
package com.example.karol.musicapp;

import android.app.Application;

import cafe.adriel.androidaudioconverter.AndroidAudioConverter;
import cafe.adriel.androidaudioconverter.callback.ILoadCallback;

public class MusicApp extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        AndroidAudioConverter.load(this, new ILoadCallback() {
            @Override
            public void onSuccess() {
            }
            @Override
            public void onFailure(Exception error) {
            }
        });
    }
}

File: app/src/main/java/com/example/karol/musicapp/MusicApp.kt
package com.example.karol.musicapp

import android.app.Application
import android.arch.persistence.room.Room
import cafe.adriel.androidaudioconverter.AndroidAudioConverter
import cafe.adriel.androidaudioconverter.callback.ILoadCallback
import com.example.karol.musicapp.Data.AppDatabase

class MusicApp : Application() {

    companion object {
        lateinit var database: AppDatabase
    }

    override fun onCreate() {
        super.onCreate()
        database =  Room.databaseBuilder(this, AppDatabase::class.java, "song.db").build()
        AndroidAudioConverter.load(this, object : ILoadCallback {
            override fun onSuccess() {}
            override fun onFailure(error: Exception) {}
        })
    }
}